src/driver.rs

```diff
@@ -292,3 +269,36 @@ pub fn main() {
         }
     }))
 }
+
+#[must_use]
+fn help_message() -> &'static str {
+    color_print::cstr!(
+"Checks a package to catch common mistakes and improve your Rust code.
+
+<green,bold>Usage</>:
+    <cyan,bold>clippy-driver</> <cyan>[OPTIONS] [--] [<<ARGS>>...]</>
+
+<green,bold>Common options:</>
+    <cyan,bold>--no-deps</>                Run Clippy only on the given crate, without linting the dependencies
+    <cyan,bold>--fix</>                    Automatically apply lint suggestions. This flag implies <cyan>--no-deps</> and <cyan>--all-targets</>
+    <cyan,bold>-h</>, <cyan,bold>--help</>               Print this message
+    <cyan,bold>-V</>, <cyan,bold>--version</>            Print version info and exit
+    <cyan,bold>--explain [LINT]</>           Print the documentation for a given lint
```

Comment:
I just tested this a bit more:

```
$ LD_LIBRARY_PATH=$(rustc --print=sysroot)/lib ./target/debug/clippy-driver -Wclippy::absurd_extreme_comparisons rustc_tools_util/src/lib.rs
error[E0601]: `main` function not found in crate `lib`
   --> rustc_tools_util/src/lib.rs:180:2
    |
180 | }
    |  ^ consider adding a `main` function to `rustc_tools_util/src/lib.rs`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0601`.
```

```
$ LD_LIBRARY_PATH=$(rustc --print=sysroot)/lib ./target/debug/clippy-driver  rustc_tools_util/src/lib.rs -- -Wclippy::absurd_extreme_comparisons
error: multiple input filenames provided (first two filenames are `rustc_tools_util/src/lib.rs` and `-Wclippy::absurd_extreme_comparisons`)
```

```
$ LD_LIBRARY_PATH=$(rustc --print=sysroot)/lib ./target/debug/clippy-driver --fix
error: Unrecognized option: 'fix'
```

1. There is no `--fix` flag on `clippy-driver` and probably some other of those flags are also not supported.
2. We don't accept `--` with additional arguments after it. This also makes the below help text wrong, which still talks about `cargo clippy`

Please rewrite the help message of the driver, so that it fits its implementation.